rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -864,6 +864,17 @@ stream {
         {{ end }}
         {{ end }}
 
+        {{ if not (empty $server.ProxySSL.CAFileName) }}
+        # PEM sha: {{ $server.ProxySSL.PemSHA }}
```

Comment:
Please use CASHA (we added this new field to differentiate the SHA of the cert and ca)

Reply:
Done